discord_slash/client.py

```diff
@@ -869,7 +869,22 @@ async def invoke_command(self, func, ctx, args):
         :param args: Args. Can be list or dict.
         """
         try:
-            await func.invoke(ctx, args)
+            # determine if args should be passed as kwargs or not
+            kwargs = False
+
+            if isinstance(args, dict):
+                for _arg_name in args.keys():
+                    if not hasattr(func, _arg_name):
+                        kwargs = False
+                        break
+                else:
+                    kwargs = True
```

Comment:
Maybe a better solution is to assume `kwargs` is true first:
```py
kwargs = True

if isinstance(args, dict):
    for _arg_name in args.keys():
        if not hasattr(func, _arg_name):
            kwargs = False
            break
else:
    kwargs = False
```

Reply:
That would end up with `kwargs` always being `False`

Reply:
No? Why will it be the case?

Reply:
Don't mind me not noticing a change in indentation lol

Reply:
Either solution seems fine tbh.